fdbserver/DataDistribution.actor.cpp

```diff
@@ -4145,10 +4147,14 @@ ACTOR Future<Void> storageServerTracker(
 					TraceEvent(SevWarn, "FailedServerRemoveKeys", self->distributorId)
 					    .detail("Server", server->id)
 					    .detail("Excluded", worstAddr.toString());
-					wait(removeKeysFromFailedServer(cx, server->id, self->lock, ddEnabledState));
-					if (BUGGIFY)
-						wait(delay(5.0));
-					self->shardsAffectedByTeamFailure->eraseServer(server->id);
```

Comment:
If the `eraseServer` method is not going to be used at all anymore, then I think it's alright to remove all occurrences of it from everywhere. It was originally created as a sort of hack for the exclude failed command in particular, but if this method doesn't need to deal with `shardsAffectedByTeamFailure` then getting rid of it completely would help clean up the code base.